### ISSUE TITLE
Content update suggestions - MBourgey, License

### ISSUE THREAD

**Issue by garantjm** [c3g/GenPipes_Docs#200]
- README and History sections mentions Mathieu Bourgey as contact, maybe it should be updated to Hector?

- [Comparative table](https://academic.oup.com/view-large/136711836?login=false) lists GNU LGPL and most Licenses links points towards [Bitbucket's GNU LGPL](https://bitbucket.org/mugqic/genpipes/src/master/COPYING.LESSER) while [License page](https://genpipes.readthedocs.io/en/latest/about/license.html) and [What is GenPipes page](https://genpipes.readthedocs.io/en/latest/get-started/what_is_genpipes.html) say GPLv3.

- Typos:

    - "So we/ll respond to all of these together:" [FAQ new user page](https://genpipes.readthedocs.io/en/latest/faq/faq_new_users.html)

    - "deployed and p re-installed on Compute Canada Servers" [FAQ C3G resource usage](https://genpipes.readthedocs.io/en/latest/faq/faq_c3g_res.html)

**Comment by garantjm** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Send edits on GenPipes readthedocs as an issue](https://trello.com/c/bGXdePaJ/5-send-edits-on-genpipes-readthedocs-as-an-issue)

**Comment by shaloo** [comment]
@garantjm 👋 hi, did you mean these or somewhere else in the docs?

1. https://genpipes.readthedocs.io/en/genpipes-v4.4.2/about/history.html#history 

[TBD: I can surely update the slide, waiting on some data from @pphector such as #downloads etc for 2019-2023.]

2. Citation: https://genpipes.readthedocs.io/en/genpipes-v4.4.2/resources/citation.html 
3. 
[TBD:  Not sure, these are for published GenPipes papers - can't change those name in citations.bib, citations.pdf and citations.txt.  Please guide @pphector what needs to be done]

Could not find Mathieu mentioned elsewhere (did a full grep), what did I miss?

LICENSE:  GenPipes is available under LGPL but OSF recommends not to use [LGPL any more for newer libraries](https://www.gnu.org/licenses/why-not-lgpl.html). 

In docs, we have explicitly mentioned that all contributions to GenPipes (by external folks) will be LGPLv3.  That is the one recommended by OSF. Should I change that to LGPL @pphector ?

**Comment by shaloo** [comment]
Others I will fix right away...

**Comment by shaloo** [comment]
Typos fixed. https://genpipes.readthedocs.io/en/genpipes-v4.4.2/search.html?q=license&check_keywords=yes&area=default and https://genpipes.readthedocs.io/en/latest/faq/faq_c3g_res.html

See all license fixes and typo updates here:  

License: https://genpipes.readthedocs.io/en/genpipes-v4.4.2/search.html?q=license&check_keywords=yes&area=default

**Comment by garantjm** [comment]
> @garantjm 👋 hi, did you mean these or somewhere else in the docs?
>
>    https://genpipes.readthedocs.io/en/genpipes-v4.4.2/about/history.html#history
>
> [TBD: I can surely update the slide, waiting on some data from @pphector such as #downloads etc for 2019-2023.]

Yes this is what I was referring to.

 And I believe GenPipes is currently deployed with a LGPLv3.

**Comment by pphector** [comment]
I agree, we should update GenPipes and the docs to use the LGPLv3 license everywhere. As for the slide, I will collect data and share with you early next week @shaloo
